exercises/grade-school/test/test_grade_school.c

```diff
@@ -171,17 +170,31 @@ static void test_grade_lists_students_in_that_grade_in_alphabetical_order(void)
    check_roster_names(&expected, &actual);
 }
 
+static void
+test_grade_returns_an_empty_list_if_there_are_no_students_in_that_grade(void)
+{
+   TEST_IGNORE();
+
+   uint8_t desired_grade = 1;
+   roster_t actual = get_grade(desired_grade);
+
+   TEST_ASSERT_EQUAL(0, actual.count);
+}
```

Comment:
From the name of this test, it seems like it should be identical to `test_grade_returns_an_empty_list_if_there_are_no_students_enrolled` except that there are students enrolled, just not in the specified grade.

Reply:
As in my reply on your first comment above, that test should be for roster, this one is for grade.
The correction of the typo for that other test should mean this test makes more sense.

However, on the whole it would be a better test if there were students in other grades in the roster when `get_grade()` is being tested for this.